docs/ens.md

```diff
@@ -0,0 +1,105 @@
+---
+id: ens
+title: Adding accounts to an ENS domain
+sidebar_label: Using ENS with DOT/KSM accounts
+---
+
+ENS (Ethereum Name Service) is a system of smart contracts on the Ethereum blockchain which allows
+users to claim domain names like `bruno.eth`. Supporting wallets can then allow senders to input ENS
+domains instead of long and unwieldy addresses. This prevents phishing, fraud, typos, and adds a
+layer of usability on top of the regular wallet user experience.
+
+> Note: You will need an ENS name and an Ethereum account with some ether in it to follow along with
+> this guide. To register an ENS name, visit the [ENS App](https://app.ens.domains) or any number of
+> subdomain registrars like [Nameth](https://nameth.io). You will also need some way to use the
+> Ethereum address - we recommend [Frame](https://frame.sh/) or [Metamask](https://metamask.io).
+
+Despite living on the Ethereum blockchain, the ENS system has multi-chain support. In this guide
+you'll go through the process of adding a KSM and DOT address to ENS. We cover both KSM and DOT to
+show two different approaches.
+
+> Note: DOT can currently only be added using the Resolver method. KSM can be added through both
+> methods described below.
+
+This guide is also available in video format [on Youtube](https://youtu.be/XKjZk-5_mQc).
+
+## Adding via the UI
+
+The [ENS App](https://app.ens.domains) allows an ENS domain owner to inspect all records bound to
+the domain, and to add new ones.
+
+![bruno.eth domain name in the ENS application](assets/ens/01-min.png)
+
+In the example above, the domain `bruno.eth` has an Ethereum and a Bitcoin address attached. Let's
+attach a KSM account. First, click the `[+]` icon in the Records tab.
+
+![The plus icon in the records tab](assets/ens/02-min.png)
+
+Then, pick "Other Addresses", "KSM", and input the Kusama address:
+
+![Inputs needed to register a KSM address](assets/ens/03-min.png)
+
+After clicking Save, your Ethereum wallet will ask you to confirm a transaction. Once processed, the
+record will show up on the domain's page:
+
+![KSM address now visible in bruno.eth records](assets/ens/04-min.png)
+
+## Adding via the Resolver
+
+DOT has only recently been added to ENS, so it's not available in the UI yet. Let's go through an
+alternative method of registering a DOT address on the `bruno.eth` domain.
+
+First, you need to find out which resolver we have set on the domain. This should be
+[Public Resolver 2](https://etherscan.io/address/0x4976fb03c32e5b8cfe2b6ccb31c09ba78ebaba41), but
+you can double check on the domain's ENS page.
+
+![The domain's resolver is highlighted on the ENS app page](assets/ens/05-min.png)
+
+Next, you need to go to the
+[resolver contract's Write Contract interface](https://etherscan.io/address/0x4976fb03c32e5b8cfe2b6ccb31c09ba78ebaba41#writeContract)
+by clicking on _Contract -> Write Contract_, or appending `#writeContract` to the resolver's URL in
```

Comment:
```suggestion
by clicking on "_Contract -> Write Contract_", or appending `#writeContract` to the resolver's URL in
```
added these in since we normally use "" for buttons